internal/translations/locales/de-DE/out.gotext.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This contents of this file seems to be duplicate of messages.gotext.json, is this intentional?